qiskit/circuit/add_control.py

```diff
@@ -146,8 +146,8 @@ def control(
                     gate.definition.data[0].operation.params[0],
                     q_control,
                     q_target[bit_indices[qargs[0]]],
-                    use_basis_gates=True,
                 )
+                continue
```

Comment:
Yes, because the decomposition is phase correct and the fragment below will change the global phase.

```python
            if gate.definition is not None and gate.definition.global_phase:
                global_phase += gate.definition.global_phase
```